services-turf/src/main/java/com/mapbox/turf/TurfMeta.java

```diff
@@ -142,6 +145,81 @@ private TurfMeta() {
     return coords;
   }
 
+  /**
```

Comment:
Here is one way to approach the implementation and to avoid unneeded ArrayList() creattion.
We have public `coorAll` methods for all Geometries. We can add private versions that take List<Point> as an extra parameter.  See the example below:

```
public static List<Point> coordAll(@NonNull Polygon polygon, boolean excludeWrapCoord) {
     return coordAll(new ArrayList<Point>(), polygon, excludeWrapCoord);
  }

  private  static List<Point> coordAll(@NonNull List<Point> coords,
                                       @NonNull Polygon polygon, boolean excludeWrapCoord) {
    int wrapShrink = excludeWrapCoord ? 1 : 0;

    for (int i = 0; i < polygon.coordinates().size(); i++) {
      for (int j = 0; j < polygon.coordinates().get(i).size() - wrapShrink; j++) {
        coords.add(polygon.coordinates().get(i).get(j));
      }
    }
    return coords;
  }
```

Reply:
Ahhhh, I see now. Got it. 

The changes above would break semver, correct? Or to preserve, could we just deprecate the current `public static List<Point> coordAll(@NonNull Polygon polygon, boolean excludeWrapCoord) {` and create a new `coordAll(@NonNull Polygon polygon, boolean excludeWrapCoord)` as you've done above?

Reply:
We are actually not breaking semver-major as only `private static` methods are added. It is a semver-minor change as new APIs are added.